sdks/java/io/clickhouse/src/test/java/org/apache/beam/sdk/io/clickhouse/AtomicInsertTest.java

```diff
@@ -92,9 +92,10 @@ public void testAtomicInsert() throws SQLException {
    * With sufficient block size, ClickHouse will atomically insert all or nothing. In the case of
    * replicated tables, it will deduplicate blocks.
    */
+  //  @Test
   @Test
   public void testIdempotentInsert() throws SQLException {
-    int size = 1000000;
+    int size = 100000;
```

Comment:
why is this smaller?

Reply:
Changed back

Reply:
setting 1M cause insert error. it also works with 500K rows, but i have changed it to 100K a deeper investigation need to be done here.